fix: improve ux for editing snapshot asset

lukicenturi opened the pull request:
Closes #(issue_number)

## Checklist

In the add asset form, in snapshot editing:
1. in the form, allow any assets to be selected,
2. if the asset already exists in the snapshot, ask the user if they want to edit that entry instead 

![image](https://user-images.githubusercontent.com/26648140/225563333-bdf72875-7742-4875-bb6b-515d9195160f.png)

kelsos commented on frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue:
```diff
@@ -441,12 +481,52 @@ const tableContainer = computed(() => {
       />
+
+      <confirm-dialog
+        max-width="700"
+        :display="!!conflictedBalanceSnapshot"
+        :title="tc('dashboard.snapshot.convert_to_edit.dialog.title')"
+        :message="tc('dashboard.snapshot.convert_to_edit.dialog.subtitle')"
+        :primary-action="
+          tc('dashboard.snapshot.convert_to_edit.dialog.actions.yes')
+        "
+        @cancel="cancelConvertToEdit"
+        @confirm="convertToEdit"
+      >
+        <v-sheet
+          v-if="conflictedBalanceSnapshot"
+          outlined
+          class="pa-4 mt-4 d-flex justify-center"
+          rounded
+        >
+          <balance-display
+            :asset="conflictedBalanceSnapshot.assetIdentifier"
+            :value="conflictedBalanceSnapshot"
+            no-icon
+          />
+          <asset-details
+            v-if="!isNft(conflictedBalanceSnapshot.assetIdentifier)"
+            :class="css.asset"
+            :asset-styled="{ padding: '2px 0.75rem' }"
+            :asset="conflictedBalanceSnapshot.assetIdentifier"
+            :opens-details="false"
+            :enable-association="false"
+          />
+          <div v-else>
+            <nft-details
+              :identifier="conflictedBalanceSnapshot.assetIdentifier"
+              :class="css.asset"
+            />
+          </div>
+        </v-sheet>
+      </confirm-dialog>
```
Do you think we can split the dialog into a new component?

kelsos reviewed (commented):
Ok so first can we do something about the double scrollbar it doesn't look good?

<img width="42" alt="image" src="https://user-images.githubusercontent.com/2269732/225915272-8fc55233-e877-4580-81d8-eab3214ecdd1.png">

I am using a 1920x1080 resolution btw.

kelsos commented:
@lukicenturi So I have removed the auto-import changes and extracted the dialog 
What remains to be done to merge this is to fix the double scrollbar

lukicenturi commented:
> @lukicenturi So I have removed the auto-import changes and extracted the dialog
> 
> What remains to be done to merge this is to fix the double scrollbar.
> 
> Make sure to pull my changes.

done

kelsos commented on frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue:
```diff
@@ -20,6 +20,8 @@ import {
 import { isNft } from '@/utils/nft';
 import { toSentenceCase } from '@/utils/text';
+import { assert } from '@/utils/assertions';
+import ConfirmSnapshotConfictReplacementDialog from '@/components/snapshots/ConfirmSnapshotConfictReplacementDialog.vue';
```
Doesn't the auto-import work in this case?
